modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-creation-and-sharing_{context}"]
+= Configuring a VPC to share within your AWS organization
+
+You can share subnets within a configured VPC with another AWS user account if that account is within your current AWS organization.
+
+.Procedure
+
+. From the AWS account that centrally manages your VPC, create or modify a VPC to your desired specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account will be the *VPC-owning AWS account*. 
```

Comment:
```suggestion
. From the AWS account that centrally manages your VPC, create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account will be the *VPC-owning AWS account*. 
```
- IBM Style advises against the use of "desired" in technical information.
- Double check the highlighting of "VPC-owning AWS account". AFAIK, we mostly use bold highlighting for UI elements.